src/main/webapp/app/admin/audits/audits.component.html

```diff
@@ -49,7 +49,7 @@ <h4 jhiTranslate="audits.filter.title">Filter by date</h4>
                     </td>
                     <td>{{ audit.type }}</td>
                     <td>
-                        <span *ngIf="audit.data?.message">{{ audit.data.message }}</span>
+                        <span *ngIf="audit.data?.sessionId">{{ audit.data.sessionId }}</span>
```

Comment:
This causes the extra data to be missing on the audits page, e.g.:
Before:
![grafik](https://user-images.githubusercontent.com/72132281/115018924-c3bc4a00-9eb8-11eb-9685-05ef95e80414.png)

After:
![grafik](https://user-images.githubusercontent.com/72132281/115018941-cb7bee80-9eb8-11eb-8e8a-b968082c2c10.png)

Reply:
Hmm it's weird but the compiler says that "message" doesn't exist. I thought it was referring to sessionId but apparently not

Reply:
Okay I removed it. Can you check if the table is still the same as before?